doc/usermanual/darkroom/modules/color/lut3d.xml

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE sect3 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+               "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+		<!ENTITY % darktable_dtd SYSTEM "../../../dtd/darktable.dtd">
+		%darktable_dtd;
+		]>
+<sect3 status="draft" id="lut3d">
+
+  <title>3D LUT</title>
+
+  <indexterm>
+    <primary>modules</primary>
+    <secondary>lut3d</secondary>
+  </indexterm>
+
+  <indexterm>
+    <primary>lut3d</primary>
+  </indexterm>
+
+  <sect4>
+
+    <title>Overview</title>
+
+    <informaltable frame="none">
+      <tgroup cols="2" colsep="0" rowsep="0">
+        <colspec colwidth="6*"/>
+        <colspec colwidth="4*"/>
+        <tbody>
+          <row>
+            <entry>
+              <para>
+                A <emphasis>3D LUT</emphasis> is a tridimensional table which allows to
+                transform any RGB value into another RGB value, normaly used for film
+                simulation and color grading.
+              </para>
+              <para>
+                The module accepts <emphasis>.cube</emphasis> and <emphasis>.png (haldclut)</emphasis> files.
+                The 3D LUT data are not saved in database nor in the XMP file, but just the
+                path inside the 3D LUT folder (see below).
```

Comment:
"The 3D LUT data are not saved in the database nor in the XMP file, only the path to the file inside the 3D LUT folder is (see below)."

Question: is it then stored in the XMP file or in the database? I'm assuming the XMP considering what comes next, but this is unclear at this point.

Reply:
The path to LUT file is saved, not the LUT data themselves. I've modified the text to make it clearer.